api/requirements.in

```diff
@@ -1,7 +1,7 @@
 # Define all requirements, only pin when necessary (and if so, add a comment explaining why).
 # See # https://git.datapunt.amsterdam.nl/Datapunt/python-best-practices/blob/master/dependency_management/
 
-django<3.0
+django
```

Comment:
General note, no changes required:

In the future a change like this could use a comment, either here or in the commit message. It kind of sails along with the current two commits, but it holds quite some significance. There was a clear reason that we needed to use <3.0 (had something to do with an outdated version of six that an external lib used).